node/util_test.ts

```diff
@@ -296,12 +296,12 @@ Deno.test("[util] format", () => {
     [10, 11],
   ], { hi: "hello" });
   const expected =
-    '[10, "hi", null, undefined, NaN, Infinity, { hello: "world" }, [10, 11], hi: "hello"]';
+    "[10, hi, null, undefined, NaN, Infinity, { hello: world }, [10, 11], hi: hello]";
   assertEquals(util.format("%o", testData), expected);
   assertEquals(util.format("%O", testData), expected);
 
   const expected2 =
-    '[10, "hi", null, undefined, NaN, Infinity, [Object], [Array], hi: "hello"]';
+    "[10, hi, null, undefined, NaN, Infinity, [Object], [Array], hi: hello]";
```

Comment:
I tested with the following Node.js script.
```js
const util = require("util");
const x = Object.assign([
  10,
  "hi",
  null,
  undefined,
  NaN,
  Infinity,
  { hello: "world" },
  [10, 11],
], { hi: "hello" });
console.log(util.format("%o", x));
```

and the result was:
```
[
  10,
  'hi',
  null,
  undefined,
  NaN,
  Infinity,
  { hello: 'world' },
  [ 10, 11, [length]: 2 ],
  [length]: 8,
  hi: 'hello'
]
```
(Node.js 16.6)

So in this case, these (`hi` and `hello`) should be quoted. (So the PR has the opposite effect)

Reply:
Oh that's true, I'll look into fixing it and adding a specific test case just for strings.

Reply:
@grian32 any chance we could include this fix for 0.107.0 next week?

Reply:
Probably @bartlomieju , I've just been busy the past week or so getting a new PC toghether.